src/modules/auth/dto/change-password.dto.ts

```diff
@@ -0,0 +1,14 @@
+// change-password.dto.ts
+import { IsNotEmpty, IsString, MinLength } from 'class-validator';
+
+export class ChangePasswordDto {
+  @IsNotEmpty()
+  @IsString()
+  @MinLength(8)
+  oldPassword: string;
+
+  @IsNotEmpty()
+  @IsString()
+  @MinLength(8)
+  newPassword: string;
```

Comment:
Add password strength validation rules... Reference the auth register endpoint for clarification.